airflow/contrib/operators/sagemaker_training_operator.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.contrib.operators.sagemaker_base_operator import SageMakerBaseOperator
+from airflow.utils.decorators import apply_defaults
+from airflow.exceptions import AirflowException
+
+
+class SageMakerTrainingOperator(SageMakerBaseOperator):
+    """
+    Initiate a SageMaker training job
+    This operator returns The ARN of the training job created in Amazon SageMaker
+
+    :param config: The configuration necessary to start a training job (templated)
+    :type config: dict
+    :param aws_conn_id: The AWS connection ID to use.
+    :type aws_conn_id: str
+    :param wait_for_completion: if the operator should block until training job finishes
+    :type wait_for_completion: bool
+    :param print_log: if the operator should print the cloudwatch log during training
+    :type print_log: bool
+    :param check_interval: if wait is set to be true, this is the time interval
+        in seconds which the operator will check the status of the training job
+    :type check_interval: int
+    :param max_ingestion_time: if wait is set to be true, the operator will fail
+        if the training job hasn't finish within the max_ingestion_time in seconds
+        (Caution: be careful to set this parameters because training can take very long)
+        Setting it to None implies no timeout.
+    :type max_ingestion_time: int
+    """
+
+    @apply_defaults
+    def __init__(self,
+                 config,
+                 aws_conn_id='aws_default',
```

Comment:
We don't need to specify this parameter here (we can keep it in the doc string, but remove it from the function signature) as it's already in the base class.

Reply:
This applies to all the other operators too.

Reply:
Updated in all operators.